bioio_sldy/sldy_image.py

```diff
@@ -259,6 +257,30 @@ def metadata(self) -> typing.Dict[str, typing.Optional[dict]]:
 
         return self._metadata
 
+    @property
+    def data_paths(self) -> typing.Set[pathlib.Path]:
+        if not self._data_paths:
+            glob_matcher = self.image_directory / f"{self._data_file_prefix}*.npy"
+            data_path_matches = self._fs.glob(f"{glob_matcher}")
+            self._data_paths = set(
+                [pathlib.Path(data_path) for data_path in data_path_matches]
+            )
+
+            if not self._data_paths:
+                self._data_paths = {
+                    self.image_directory / filename
+                    for filename in os.listdir(f"{self.image_directory}/")
```

Comment:
is this trailing slash always ok?

Reply:
Nope! I meant to use pathlib for this, nice catch